src/Routes/ImageManagerDetail/ImageDetailTabs.js

```diff
@@ -34,7 +34,7 @@ const ImageDetailTabs = ({
     splitUrl[paramIndex] = selectedTab;
 
     if (selectedTab === 'details') {
-      history.push(splitUrl.splice(0, 5).join('/'));
+      history.push(splitUrl.splice(0, 6).join('/'));
```

Comment:
just a thought. maybe instead of using an array and magic numbers to get the specific value or to generate a new url path. create an object that split a url and put each field in an attribute so it will be easy to read the code later?

It's just a suggestion.